draft-ietf-scitt-architecture.md

```diff
@@ -149,17 +149,18 @@ Issuers can register their Signed Statements on any Transparency Service, with t
 This document describes the scalable, flexible, and decentralized SCITT architecture.
 Its goal is to enhance auditability and accountability across supply chains.
 
-In supply chains, artifacts travel down the chain until they are eventually consumed by someone.
-Consumers like to have information about the artifacts that they consume.
+In supply chains, downstream artifacts are built upon upstream artifacts.
+The complexity of traceability and quality control for these supply chains increases with the number of artifacts and parties contributing to them.
+All parties in supply chains like information about these artifacts.
 There are many parties who publish information about artifacts:
 For example, the original manufacturer may provide information about the state of the artifact when it left the factory.
 The shipping company may add information about the transport environment of the artifact.
 Compliance auditors may provide information about their compliance assessment of the artifact.
 Security companies may publish vulnerability information about an artifact.
-Consumers may even publish the fact that they consume an artifact.
+Some of these parties may publish information about their analysis or use of an artifact.
 
-SCITT provides a way for consumers to obtain this information in a way that is "transparent", that is, parties cannot lie about the information that they publish without it being detected.
-SCITT achieves this by having producers publish information in a Transparency Service, where consumers (also called Verifiers) can check the information.
+SCITT provides a way for these parties to obtain this information in a way that is "transparent", that is, parties cannot lie about the information that they publish without it being detected.
+SCITT achieves this by having producers publish information in a Transparency Service, where Verifiers can check the information.
```

Comment:
I could go either way: "Verifiers", or "Relying Parties". Choose to merge/keep what you'd like in this PR.
```suggestion
SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.
```

Reply:
Ditto with above, I did not mean to change Verifiers or other entity to be Client or Relying Party (that is really another PR), but to say parties in line with other usages of it like actor/stakeholder that was previously existing in other parts of the introduction.